app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
The information in this file doesn't belong on a remote repo. They are considered secrets and shouldn't be made publicly available.